server/src/test/scala/apis/ReCaptchaApiSpec.scala

```diff
@@ -0,0 +1,52 @@
+package apis
```

Comment:
Let's the same package than the one used by the class being tested, `net.wiringbits.apis`